assets/dev/js/editor/elements/views/behaviors/inline-editing.js

```diff
@@ -203,6 +203,7 @@ InlineEditingBehavior = Marionette.Behavior.extend( {
 
 		// Is it repeater?
 		if ( 3 === parts.length ) {
+			container = container.repeaters[ parts[ 0 ] ];
```

Comment:
maybe replace for readability 

		const isRepeater = 3 === parts.length;
		const getContainerFromRepeater = ( containerWithRepeater ) => {
			const repeater = containerWithRepeater.repeaters[ parts[ 0 ] ];
			return repeater.children[ parts[ 1 ] ];
		};

		if ( isRepeater ) {
			container = getContainerFromRepeater( container );
			key = parts[ 2 ];
		}

Reply:
@TzviRabinovitch I refactored a bit to make code clearer, more about using clearer variable names to make clear what we are doing than anything else, what do you think?